scripts/lite/multisig.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+#CARDANO_NODE_SOCKET_PATH=/home/jordan/Repos/Work/cardano-haskell/cardano-node/shelleytest/socket/node-1-socket
```

Comment:
Make sure you remember to remove this:

```suggestion
```

Reply:
This was a test script that I have removed.